content/en/data_streams/manual_instrumentation.md

```diff
@@ -94,6 +94,69 @@ set_consume_checkpoint(
 "<datastream-type>", "<datastream-name>", headers.get
 )
 
+{{< /code-block >}}
+{{% /tab %}}
+{{% tab ".Net" %}}
+<div class="alert alert-warning">
+  <strong>Note:</strong> DSM instrumentation does not work in Async operations
```

Comment:
I wonder if this wording is not too strong... It will work, it's just that context is broken/lost on a `send` following a `receive`. I don't know how to convey that in a clear way.

Reply:
How does:

`Note: In async operations, this may not work as expected because the context derived from the incoming message can be lost when producing a new message in different threads.`

Sound?